Tests/CloudEnvironmentTests/resources/mappings.json

```diff
@@ -176,5 +176,11 @@
          "file:Tests/CloudEnvironmentTests/resources/config_cf_example.json"
        ]
    }
+  "HyperProtectDBaaS": {
```

Comment:
Missing a comma above this line (invalid JSON, which is I suspect why the tests are failing)